plugins/app-rating/README.md

```diff
@@ -0,0 +1,43 @@
+# app-rating
+
+Welcome to the app-rating plugin!
+
+_This plugin was created through the Backstage CLI_
+
+This plugin allows your users to rate the Backstage application.
+
+## Getting started
+
+Your plugin has been added to the example app in this repository, meaning you'll be able to access it by running `yarn start` in the root directory, and then navigating to [/app-rating](http://localhost:3000/app-rating).
+
+You can also serve the plugin in isolation by running `yarn start` in the plugin directory.
+This method of serving the plugin provides quicker iteration speed and a faster startup and hot reloads.
+It is only meant for local development, and the setup for it can be found inside the [/dev](./dev) directory.
+
+## Install backend
+
+You need to install app-rating-backend to persist ratings.
+
+`yarn add --cwd packages/backend @backstage/plugin-app-rating-bakcend`
```

Comment:
I think there's a typo here but the general pattern has been to link to the backend documentation instead and it helps avoid duplication

Reply:
True